### PR TITLE
Added stylelintignore for generated coverage files.

### PR DESCRIPTION
**Pull request by krishnaglick**
Added stylelintignore for generated coverage files.

### CODE REVIEW

**Review by teallarson — APPROVED** [review]
confirmed that this does fix the local OSS build

File: airbyte-webapp/.stylelintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+coverage/**/*.css
```
Comment (ambirdsall) [inline airbyte-webapp/.stylelintignore:1]: nit of all nits, but could you put a final newline in this file? `.editorconfig` is agnostic except for python files, where `insert_final_newline = true` is explicitly set, but our files nearly universally have them. Perhaps most importantly, though, is that I think github's big red "no final newline here" icon is kind of ugly :shrug: :sweat_smile:

Comment (ambirdsall) [reply airbyte-webapp/.stylelintignore:1]: Is there a reason to scope this specifically to `*.css` files? IIUC, everything in `coverage/` is autogenerated, so why not ignore `coverage/**/*`?

**Review by ambirdsall — APPROVED** [review]
Minor suggestions, but this improves our setup with no apparent downsides :raised_hands: :sparkles: :rocket: :guitar:

File: airbyte-webapp/.stylelintignore (hunk 1 above)
Comment (krishnaglick) [reply airbyte-webapp/.stylelintignore]: I used the inline editor, I think it got the newline

**Review by ambirdsall — APPROVED** [review]
:sparkles: :guitar: :shipit: :guitar: :sparkles:

**Pull request merged**